eksctl/carbonplan.jsonnet

```
@@ -60,12 +69,12 @@ cluster {
                 "hub.jupyter.org/dedicated": "user:NoSchedule"
             },
 
-        } + n for n in nodes
+        } + n for n in notebookNodes
```

I don't know jsonnet, but there is an inconsistency here.

Below it sais `} for n in daskNodes`, but here it sais `} + n for n in notebookNodes`. I'll go ahead and guess this is wrong, but perhaps its the other that is wrong? No clue.

```suggestion
        } n for n in notebookNodes
```

---

@consideRatio yeah, I was trying to be 'clever' and just be able to set `instanceType` for both non-spot and spot node pools. However, this becomes unmaintainable soon, so I've reverted that. Now the config is the same as what you'll pass to eksctl.